<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FighterManagerDlg</class>
 <widget class="QDialog" name="FighterManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Fighters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ipponboard.qrc">
    <normaloff>:/res/icons/category.png</normaloff>:/res/icons/category.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="label_filterinfo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget_fighters">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_import">
       <property name="toolTip">
        <string>Move fighter up</string>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
       <property name="icon">
        <iconset resource="ipponboard.qrc">
         <normaloff>:/res/icons/import_list.png</normaloff>:/res/icons/import_list.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_export">
       <property name="toolTip">
        <string>Move fighter down</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
       <property name="icon">
        <iconset resource="ipponboard.qrc">
         <normaloff>:/res/icons/export_list.png</normaloff>:/res/icons/export_list.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_settings">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Settings for import/export</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ipponboard.qrc">
         <normaloff>:/res/icons/settings.png</normaloff>:/res/icons/settings.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_add">
       <property name="toolTip">
        <string>Add a new fighter</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="ipponboard.qrc">
         <normaloff>:/res/icons/add.png</normaloff>:/res/icons/add.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_remove">
       <property name="toolTip">
        <string>Remove the currently selected weight class</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="ipponboard.qrc">
         <normaloff>:/res/icons/delete.png</normaloff>:/res/icons/delete.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ipponboard.qrc"/>
  <include location="ipponboard.qrc"/>
 </resources>
 <connections/>
</ui>
